Common/Data/Custom/IconicTypes/IndexedLinkedData2.cs

```diff
@@ -0,0 +1,149 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using NodaTime;
+using QuantConnect.Data;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using ProtoBuf;
+
+namespace QuantConnect.Data.Custom.IconicTypes 
+{
+    /// <summary>
+    /// Data type that is indexed, i.e. a file that points to another file containing the contents
+    /// we're looking for in a Symbol.
+    /// </summary>
+    [ProtoContract(SkipConstructor = true)]
+    public class IndexedLinkedData2 : IndexedBaseData
```

Comment:
What's the difference with `IndexedLinkedData`?

Reply:
`IndexedLinkedData2` throws whenever `GetSource` is called with `isLiveMode = true`,  whereas the other doesn't. This behavior is required to keep parity with some tests that were previously Benzinga/Tiingo news data sources.